src/components/Garden/Feed/ProposalCard.tsx

```diff
@@ -40,6 +48,10 @@ function Card({ loading = false, proposal }) {
     history.push(path)
   }, [history, proposal.number, proposal.type])
 
+  const handleViewProfile = useCallback(() => {
+    history.push(`/profile?account=${account}`)
```

Comment:
Here we want to view the `proposal.creator` profile instead. Otherwise, we are always redirecting to the connected account. And raise an error, if no account connected.

Reply:
Yes of course. I will make those changes.

Reply:
Just made those changes @0xGabi